Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: bootstrap
```

Comment:
Is there a more robust way of ensuring a clean virtual environment and Python3?
(And in general, what would be the best practice for ensuring the desired environment?)